src/routes/_surfaces/assets/TableAssets.svelte

```diff
@@ -67,6 +74,14 @@
       tableName
     );
   };
+
+  const deleteSource = (tableName: string) => {
+    deleteSourceApi(
+      tableName,
+      $persistentModelStore.entities,
+      $derivedModelStore.entities
+    );
+  };
```

Comment:
vscode tells me that `deleteSourceApi` only needs one argument, has 3. Fixed it, which also removed th need for one import

Reply:
Not sure why the build doesnt fail because of this